projects/plutus/inc/ltc_afe.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+#include "gpio.h"
+#include "spi.h"
+#include "status.h"
+
+// number of devices in daisy chain (including master)
+#define LTC_AFE_DEVICES_IN_CHAIN                1
+
+#define LTC_AFE_CELLS_IN_REG                    3
+#define LTC_AFE_GPIOS_IN_REG                    3
+#define LTC_CELLS_PER_DEVICE                    12
+
+typedef enum {
+  LTC_AFE_DISCHARGE_TIMEOUT_DISABLED = 0,
+  LTC_AFE_DISCHARGE_TIMEOUT_30_S,
+  LTC_AFE_DISCHARGE_TIMEOUT_1_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_2_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_3_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_4_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_5_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_10_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_15_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_20_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_30_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_40_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_60_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_75_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_90_MIN,
+  LTC_AFE_DISCHARGE_TIMEOUT_120_MIN
+} LTCAFEDischargeTimeout;
+
+typedef enum {
+  LTC_AFE_ADC_MODE_27KHZ = 0,
+  LTC_AFE_ADC_MODE_7KHZ,
+  LTC_AFE_ADC_MODE_26HZ,
+  LTC_AFE_ADC_MODE_14KHZ,
+  LTC_AFE_ADC_MODE_3KHZ,
+  LTC_AFE_ADC_MODE_2KHZ,
+  NUM_LTC_AFE_ADC_MODE
+} LTCAFEADCMode;
```

Comment:
Added a comment about the tradeoffs you make here between speed and noise. Made a reference to the timing estimates and noise calculations in the datasheet